README.md

```diff
@@ -266,11 +266,11 @@ In order for the app to function correctly, the user must set up their own envir
 
 create a .env file that includes the following:
     
-    *  DATABASE_URL=postgres://postgres:lambdaschoolgroa@groadb-dev.cbayt2opbptw.us-east-1.rds.amazonaws.com:5432/postgres
-    *  TESTING_DB_URL=postgres://@localhost:5432/postgres
-    *  RECOMMENDATION_URL=http://a13327d835de211ea92c80a488b922f7-342789911.us-east-1.elb.amazonaws.com/movie-recommender
+    *  DATABASE_URL
+    *  TESTING_DB_URL
+    *  RECOMMENDATION_URL
     *  JWT_SECRET 
-    *  TOKEN_EXP=8h
+    *  TOKEN_EXP
```

Comment:
These should also state where to get the information from, and why they are important to running the application locally.

Reply:
Added instructions